ui/app/components/clients/history.js

```diff
@@ -253,12 +261,27 @@ export default class History extends Component {
       .mounts?.find((mount) => mount.label === auth);
   }
 
+  get filteredActivityByMonth() {
+    const namespace = this.selectedNamespace;
+    const auth = this.selectedAuthMethod;
+    const namespaceData = this.getActivityResponse?.byMonth.map((m) => m.namespaces_by_key[namespace]);
```

Comment:
Since namespace can be null if unset, I would move this line below line 270 so you don't get weird data or a failure when namespace is null

Reply:
Good call! I even had it there and then during "cleanup" moved all my variables to the top of the method 🤦